src/discord.ts

```diff
@@ -37,30 +31,46 @@ export default class Discord {
       });
   }
 
-  getConnectedApp(discordServerId: string): Promise<string> {
+  getConnectedApp(appId: string, discordServerId: string): Promise<string> {
```

Comment:
It seems a bit strange to have to put the appId in order to know the app. 🤔

Reply:
Right, I've updated the API like so => https://bitbucket.org/comcomai/nft-server/pull-requests/39/use-controllers-fix-file-name-convention#Lsrc/discord/discord.routes.tsT40
With the master key, you can just enter an empty string and get any apps connected to any servers.
With an access key to a specific app, it would be more natural to call an app getter API (not yet implemented) instead of this. With this API, as you pointed out, the admin will need to know the appId prior to the API call. If the admin doesn't have access to the app, it will throw an unauthorized error.

Reply:
@liayoo would it be applicable to move the appId param after the discordServerId param and giving a default value of empty string?

Reply:
Great idea! Done.